src/index.tsx

```diff
@@ -4,13 +4,16 @@ import "./index.css";
 import App from "./App";
 import { BrowserRouter as Router } from "react-router-dom";
 import { AuthProvider } from "./context/Auth/auth-context";
+import { QuizProvider } from "./context/Quiz/quiz-context";
```

Comment:
You can create an index file in Auth folder and export all your providers from there.  Will make you code looks clean. Either way is cool.